docs/schedule/process.zh_TW.md

```diff
@@ -0,0 +1,337 @@
+   - 在要求投票之前，至少有一個GTFS製片人和一個GTFS消費者應實施提議的變更。預計GTFS生產者包括面向公眾的變化GTFS在拉取請求評論中提供並提供指向該數據的鏈接，並且GTFS消費者在 pull request 評論中提供了一個指向應用程序的鏈接，該應用程序正在以非平凡的方式利用更改（即，它支持新的或改進的功能）。
+1. 投票持續的最短時間足以覆蓋 7FULL日曆日和 5FULL瑞士工作日。投票於世界標準時間 23:59:59 結束。
+   - 倡導者應公佈具體endtime在start的投票。
+   - 在投票期間，只允許對提案進行編輯性修改（錯別字、措辭可以更改，只要不改變意思）。
+   - 任何人都可以對拉取請求以評論的形式投票是/否，並且投票可以更改，直到end的投票期。
+   如果選民改變她的投票，建議通過刪除投票並寫新投票來更新原始投票評論來做到這一點。
+   - 之前的投票start的投票期不予考慮。
+   - 投票開始和結束必須在 上宣布 [更改郵件列表](https://groups.google.com/forum/#!forum/gtfs-changes)。
+1. 如果至少有 3 票一致同意，則該提案被接受。
+   - 提議者的投票不計入 3 票總數。例如，如果提議者 X 創建一個拉取請求並投贊成票，而用戶 Y 和 Z 投贊成票，則這計為總共 2 票贊成票。
+   - 反對票應有動機，並最好提供可操作的反饋。
+   - 如果投票失敗，那麼倡導者可以選擇繼續對該提案進行工作，或者放棄該提案。
+   辯護人的任何一項決定都必須在 中宣布 [更改郵件列表](https://groups.google.com/forum/#!forum/gtfs-changes)。
+   - 如果倡導者繼續提案工作，則可以在任何時候要求進行新的投票time.
+1. 任何在 30 個日曆日內保持非活動狀態的拉取請求都將被關閉。當拉取請求關閉時，相應的提案被視為已放棄。倡導者可以隨時重新打開拉取請求time如果他們希望繼續或維持對話。
+1. 如果提案被接受：
+   - Google 承諾合併投票通過的拉取請求版本（前提是貢獻者已簽署[CLA](../CONTRIBUTING.md) ），並在 5 個工作日內執行拉取請求。
+   - 翻譯不得包含在原始拉取請求中。
+   Google 負責最終將相關翻譯更新為受支持的語言，但純Translation歡迎來自社區的拉取請求，並在所有編輯評論得到解決後立即接受。
+1. 拉取請求的最終結果（接受或放棄）應在最初宣布拉取請求的同一 Google 網上論壇線程中公佈。
+
+<hr/>
+
+## 指導原則
+
+為了保留原有的視野GTFS , 在擴展規範時要考慮一些指導原則：
+
+**提要應該易於創建和編輯**<br/>我們選擇 CSV 作為規範的基礎，因為使用電子表格程序可以輕鬆查看和編輯它，並且text編輯器，這對較小的機構很有幫助。它也很容易從大多數編程語言和數據庫中生成，這對大型提要的發布者來說非常有用。<br/>
+
+**提要應該易於解析**<br/>提要閱讀器應該能夠以盡可能少的工作提取他們正在尋找的信息。對 feed 的更改和添加應該盡可能廣泛有用，以最大限度地減少 feed 讀者需要實現的代碼路徑的數量。 （但是，應該優先考慮讓創建更容易，因為最終將有更多的提要發布者而不是提要閱讀者。）<br/>
+
+**規範是關於乘客信息的**<br/>GTFS主要與乘客信息有關。也就是說，規範首先應包括有助於為騎手提供動力工具的信息。運輸機構可能希望在系統之間內部傳輸大量面向運營的信息。GTFS不用於該目的，並且可能存在其他可能更合適的面向操作的數據標準。<br/>
+
+**對規範的更改應該向後兼容**<br/>在向規範添加功能時，我們希望避免進行會使現有提要無效的更改。在現有的提要發布者想要為其提要添加功能之前，我們不想為他們創造更多的工作。此外，只要有可能，我們希望現有的解析器能夠繼續讀取較新提要的較舊部分。<br/>
+
+**不鼓勵投機功能**<br/>每一項新功能都增加了訂閱源的創建和閱讀的複雜性。因此，我們要注意只添加我們知道有用的功能。理想情況下，任何提案都將通過為使用新功能的真實交通系統生成數據並編寫軟件來讀取和顯示來進行測試。請注意，GTFS允許通過添加官方解析器和驗證器忽略的額外列和文件來擴展格式，因此可以輕鬆地對提案進行原型設計並在現有提要上進行測試。<br/>
+
+<hr/>
+
+## 修訂記錄
+
+**2021 年 9 月 15 日**<br/>
+
+* 允許收費站 (pathway_mode=6) 是雙向的。見[討論](https://github.com/google/transit/pull/276)。
+
+**2021 年 9 月 13 日**<br/>
+
+* 更新`stop_name`最佳實踐。見[討論](https://github.com/google/transit/pull/282)。
+
+**2021 年 8 月 27 日**<br/>
+
+* 更新GTFS Schedule到[RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119) 。見[討論](https://github.com/google/transit/pull/277)。
+
+**2021 年 1 月 4 日**<br/>
+
+* 澄清了`stop_times. stop_id`stop_id .見[討論](https://github.com/google/transit/pull/258)。
+* 定義的正和非零字段符號。見[討論](https://github.com/google/transit/pull/251)。
+
+**2020 年 10 月 2 日**<br/>
+
+* 將`frequencies.headway_secs`的字段類型從非負數更改為正整數。見[討論](https://github.com/google/transit/pull/249)。
+
+**2020 年 5 月 25 日**<br/>
+
+* 定義pathways.txt ,levels.txt和attributions.txt作為可翻譯的表格。添加了翻譯多語言`signposted_as`值的建議。見[討論](https://github.com/google/transit/pull/220)。
+
+**2020 年 5 月 13 日**<br/>
+
+* 添加了`continuous_pickup`和`continuous_drop_off`到routes.txt和stop_times.txt .改變了shape_id從“可選”到“有條件地需要”。見[討論](https://github.com/google/transit/pull/208)。
```

Comment:
routes.txt, stop_times.txt, shape_id not code
Optional and conditionally required translated